helm/cas-ciip-portal/templates/secret.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     "helm.sh/hook-delete-policy": "before-hook-creation"
 type: Opaque
 data:
+  session-secret: {{ randAlphaNum 32 | b64enc | quote }}
```

Comment:
Since this secret template is scheduled as a helm `pre-install` hook, the change to add a `session-secret` field will need to be manually patched in dev, test, and prod before releasing this patch or things will go boom.